modules/custom/az_migration/az_migration.info.yml

```diff
@@ -6,6 +6,7 @@ package: 'The University of Arizona - Experimental'
 lifecycle: experimental
 lifecycle_link: 'https://github.com/az-digital/az_quickstart/blob/main/RELEASES.md#experimental-features'
 dependencies:
+  - drupal:migmag
```

Comment:
Note: this is optional since there is already a dependency on `media_migration` https://git.drupalcode.org/project/media_migration/-/blob/8.x-1.x/media_migration.info.yml?ref_type=heads